### ISSUE TITLE
Support comma delimited string for namedtuple.

### ISSUE THREAD

**Issue by graingert** [python/mypy#1518]
```python
from collections import namedtuple

X = namedtuple('X', 'a, b')
x = X(1, 'x')
x = X(b='y', a='x')  # error: Unexpected keyword argument "a" for "X"
x.a + [1]  # error: "X" has no attribute "a"
class Y(X): 
    """you can use a namedtuple class as a base class"""
```

**Comment by The-Compiler** [comment]
#529 is the same thing for `enum.Enum`.

**Comment by elazarg** [comment]
I will add it to PR #1810, although it's easy enough not to wait for it. It requires a single line change [here](../../mypy/blob/master/mypy/semanal.py#L1642). Changing `items = str_expr.value.split()` into `items = str_expr.value.replace(',', ' ').split()`

However, as discussed in the `Enum` case, it's a good idea not to use this form.

**Comment by elazarg** [comment]
This has been be fixed by #1810
